src/py/flwr/common/logger.py

```diff
@@ -125,6 +126,22 @@ def update_console_handler(
 console_handler.setLevel(logging.INFO)
 FLOWER_LOGGER.addHandler(console_handler)
 
+# Change log level via env (always get timestamp)
+if log_level := os.getenv("PYTHONLOGLEVEL"):
+    try:
+        use_time_stamps = log_level.upper() == "DEBUG"
+        update_console_handler(
+            level=log_level, timestamps=use_time_stamps, colored=True
+        )
+    except Exception:  # pylint: disable=broad-exception-caught
+        # Alert user but don't raise exception
+        FLOWER_LOGGER.log(
+            ERROR,
+            "Failed to set logging level %s. Using default level: %s",
+            log_level,
+            logging.getLevelName(console_handler.level),
+        )
```

Comment:
Should we move all the code to the bottom of this file, and then jus use `log` function to log this message? Wdyt?
```suggestion
       log(
            ERROR,
            "Failed to set logging level %s. Using default level: %s",
            log_level,
            logging.getLevelName(console_handler.level),
        )
```